src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -130,9 +130,17 @@ As a **Hero or Sidekick**, make sure you attend to the following throughout your
 * Don't reference internal people by name. If they have a GH account, @mention their GH handle. If they don't, talk instead about teams ("talk to a browser team engineer" or "Support Engineer") rather than people.
 * You can mention the #documentation channel and hero.
 
-## Merge from develop into main work (or, when do we publish?) [#merging-main]
-The Hero currently merges three times a day: At 9 AM, noon, and 3 PM Pacific.
-* Slackbot will remind us about this in #docs_deploys.
+## Merge releases into main work (or, when do we publish?) [#merging-main]
+The Hero currently merges three times a day: At 9 AM (morning), 12 PM (noon), and 3 PM (afternoon) Pacific. We merge release branches into develop to avoid interuptions when someone merges into develop during a release. Read more about this workflow [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+To start a release:
```

Comment:
In general, I'd like this procedure to be more focused. I'd like to pick a single method as our team's preferred method and pull out the other possibilities. 

It's ok if people do this in a slightly different way, but I think we should be opinionated about the One True Way :)

Also, although I think it's good to mention that the "daily-release*" are protected branches, I'd pull that info out of the procedure too as well as a little info about what a protected branch is, what that means for us.